brand/migrations/0003_remove_commentary_recommended_order_and_more.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.5 on 2022-06-12 13:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('brand', '0002_commentary_our_take'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='commentary',
+            name='recommended_order',
+        ),
+        migrations.AddField(
+            model_name='commentary',
```

Comment:
oh lord. I'd forgotten all about this field. As long as it's here, it should really be a choice field, though I'd much prefer to just remove it.

Reply:
Pushed a fix for this